aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -673,6 +673,24 @@ void huber_kernel(TensorIterator& iter, double delta) {
 
 void sigmoid_backward_kernel(TensorIterator& iter) {
   AT_DISPATCH_FLOATING_TYPES_AND2(kBFloat16, kHalf, iter.dtype(), "sigmoid_backward_cpu", [&]() {
+    if (iter.dtype() == kBFloat16) {
+      auto one_vec = Vectorized<float>((float)(1));
+      cpu_kernel_vec(iter,
+        [=](BFloat16 a, BFloat16 b) -> BFloat16 {
+          float a0 = static_cast<float>(a);
+          float b0 = static_cast<float>(b);
+          return a0 * (float(1) - b0) * b0;
+        },
+        [=](Vectorized<BFloat16> a, Vectorized<BFloat16> b) {
+          Vectorized<float> a0, a1, b0, b1;
```

Comment:
This code looks like good candidate to have inside of vec library

Reply:
Fixed!